Added round button component definition (#1980)
IDEMSInternational/parenting-app-ui: documentation/docs/components/round_button.md

# Round button

> A round button is a graphical user interface (GUI) component that has a circular appearance and functions as an interactive element. It is often used to trigger actions, navigate through different screens or menus, or initiate specific operations within the app. The appearance of round buttons can vary depending on the app's design guidelines or the preferences of the app developer. It can have different colors, sizes, and even icons or text within them to provide visual cues about their functionality.

Reviewer: ```suggestion
A round button is an interactive element with a circular appearance. It is used to trigger actions, such as navigating through different screens or menus, or initiating specific operations within the app. Round buttons can have different colours, sizes, and even icons within them to provide visual cues about their functionality.
```

Author: Looks great!
